core/dbt/exceptions.py

```diff
@@ -236,7 +236,7 @@ class VersionsNotCompatibleException(SemverException):
 
 
 class NotImplementedException(Exception):
-    pass
+    MESSAGE = "ERROR: this is not implemented"
```

Comment:
```suggestion
    MESSAGE = 'Not Implemented'
```

Reply:
this is pretty minor, but let's make the `MESSAGE` attribute look more like the other MESSAGEs defined for dbt's exceptions.

I think we're good to merge this if you accept this change!

Reply:
(comment edited, see above)

Reply:
Got it! I clicked "commit suggestion" on GitHub web, I believe this made the change.